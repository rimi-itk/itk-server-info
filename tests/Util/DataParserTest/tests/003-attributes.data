--- vhost file="/etc/apache2/sites-enabled/test.conf" type="apache" start
ServerName test.example.com
DocumentRoot /data/www/example/htdocs/
--- vhost end
