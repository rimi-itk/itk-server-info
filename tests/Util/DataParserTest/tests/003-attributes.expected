vhost:
  '@text': |
    ServerName test.example.com
    DocumentRoot /data/www/example/htdocs/
  '@attributes':
    file: '/etc/apache2/sites-enabled/test.conf'
    type: 'apache'
